libs/sdk-core/src/breez_services.rs

```diff
@@ -730,6 +730,18 @@ impl BreezServices {
             .btc_receive_swapper
             .create_swap_address(channel_opening_fees)
             .await?;
+
+        if let Some(webhook_url) = self.persister.get_webhook_url()? {
```

Comment:
If this is part of the btc_receive_swapper and always executed as part of creating swap address perhaps this logic should be inside the btc_receive_swapper?

Reply:
Good point, I moved it in `create_swap_address`: c35fb43b6b168fa76b7091465eefee2fae289b7f